### PR TITLE
Merge develop to master for v3.0.8

### PR DESCRIPTION
**Pull request by chchan-mdsol**

### CODE REVIEW

**Comment by linvald** [comment]
Shouldnt this be a MINOR version update?
```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.```

**Comment by chchan-mdsol** [comment]
I wrote the title of this PR based on the version written in the changelog. However I agree that a minor version update sounds more appropriate

File: README.rdoc [hunk 1]
```diff
@@ -163,6 +164,39 @@ Rails.application.config.session_store ActionDispatch::Session::ActiveModelMemca
...
+CASClient::Frameworks::Rails::Filter.configure(
+  ...
+    ticket_store: :active_model_redis_ticket_store,
+    ticket_store_config: {
+        redis_settings: REDIS_SETTINGS
+    }
+    ...
+)
+
+Set up an ActiveModelRedisStore to be able to have the session be created and destroyed with the service_tickets under <tt>config/initializers/active_model_redis_store.rb<tt>; similar to the one in templates/active_model_redis_store.rb
```
Comment (pchavoshi-mdsol) [inline README.rdoc:180]: didnt close <tt> tag so formatting is broken

**Review by pchavoshi-mdsol — APPROVED** [review]

**Review by linvald — APPROVED** [review]

File: README.rdoc (hunk 1 above)
Comment (JeremyAtMedidata) [reply README.rdoc]: Recent update:
- changed CHANGELOG version to 3.1.0
- added missing closing tag in README

**Pull request merged**